src/main/resources/config/application.yml

```diff
@@ -154,8 +154,7 @@ application:
         registration-captcha-period-seconds: 21600
     client-grant-types: client_credentials, refresh_token, password, tfa_otp_token
     client-scope: openid
-    default-client-id: internal
-    default-client-secret: internal
+    tenant-path: /config/tenants/XM/tenant-config.yml
```

Comment:
remove this line

Reply:
default-client-id must be **internal**
we need possibility to change secret only